src/handlers/ground-truths/prompts.ts

```diff
@@ -0,0 +1,57 @@
+import { GroundTruthsSystemMessageTemplate, ModelApplications } from "../../types/llm";
+
+const CODE_REVIEW_GROUND_TRUTHS_SYSTEM_MESSAGE = {
```

Comment:
This can be ignored as it's only relevant to #11 so this is all the work on it I'll do here. 

lmk if it's better to remove all #11 related logic and I will.